client/src/components/DataFiles/DataFilesListing/DataFilesListing.js

```diff
@@ -89,12 +90,17 @@ const DataFilesListing = ({ api, scheme, system, path }) => {
       width: 0.5,
       Cell: fileNavCellCallback
     },
-    { Header: 'Size', accessor: 'length', Cell: FileLengthCell, width: 0.2 },
+    { Header: 'Size', accessor: 'length', Cell: FileLengthCell, width: 0.15 },
     {
       Header: 'Last Modified',
       accessor: 'lastModified',
       Cell: LastModifiedCell,
-      width: 0.2
+      width: 0.15
+    },
+    {
+      Header: 'Path',
+      width: 0.1,
+      Cell: el => <ViewPathCell file={el.row.original} />
```

Comment:
Need to confirm with Tracy, but this may need to be a configurable option. If so, we can work with the solution from PR https://github.com/TACC/Frontera-Portal/pull/279

Reply:
Yes, this needs to be configurable. I'm going to wait for master to be updated with 279